Xamarin.Forms.Platform.UAP/FontExtensions.cs

```diff
@@ -111,10 +111,10 @@ static IEnumerable<string> GetAllFontPossibilities(string fontFamily)
 			var (hasFontAlias, fontPostScriptName) = FontRegistrar.HasFont(fontFamily);
 			if (hasFontAlias)
 			{
+				var fontName = FontRegistrar.GetFontName(fontFamily);
 				var file = FontFile.FromString(IOPath.GetFileName(fontPostScriptName));
-				var formated = $"{fontPostScriptName}#{file.GetPostScriptNameWithSpaces()}";
-				yield return formated;
-				yield return fontFamily;
```

Comment:
Note: this line was not needed when the font was found in the FontRegistrar. In fact, it caused problems when setting FontFamily in a CanvasTextFormat object (see FontImageSourceHandler.cs)